include/mbedtls/config.h

```diff
@@ -2798,7 +2798,30 @@
 //#define MBEDTLS_SSL_CACHE_DEFAULT_MAX_ENTRIES      50 /**< Maximum entries in cache */
 
 /* SSL options */
-//#define MBEDTLS_SSL_MAX_CONTENT_LEN             16384 /**< Maxium fragment length in bytes, determines the size of each of the two internal I/O buffers */
+//#define MBEDTLS_SSL_MAX_CONTENT_LEN             16384 /**< Maximum fragment length in bytes, determines the size of each of the two internal I/O buffers */
```

Comment:
Style: these lines would be better wrapped at 80 columns. We break this rule elsewhere in the source code, but try not to.

Reply:
I think the reason this is done for many items in config.h is that splitting a line which was commented with `//` is awkward, ie

```
//#define MBEDTLS_SSL_MAX_CONTENT_LEN             16384 /**< Maximum fragment length in bytes, determines the size of each
                                                             of the two internal I/O buffers */
```
... is a compiler error, and if you add a `//` on the second line then it breaks when you uncomment the macro if you don't also uncomment the second line.

Suggest that if you want to break this pattern then it'd be best to do it in a single change which reformats all config.h items to get the desired layout throughout (there a lot like this one, or with even longer inline comments after them).